api/services/prompt_service.py

```diff
@@ -98,7 +98,7 @@ def fetch_prompt_response(self, prompt):
         return (
             split_prompt_by_underscore[1]
             if len(split_prompt_by_underscore) > 1
-            else None
+            else split_prompt_by_underscore[0]
```

Comment:
As now only simple name will be used. So the prompt response will have no underscore